src/test/java/core/basesyntax/service/strategy/ReduceQuantityStrategyTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.service.strategy;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.dto.FruitDto;
+import core.basesyntax.model.Fruit;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReduceQuantityStrategyTest {
+    private static OperationStrategy strategy;
+
+    @BeforeClass
+    public static void beforeClass() {
+        strategy = new ReduceQuantityStrategy(new FruitDaoImpl());
+    }
+
+    @Test
+    public void test_reduceProcess_Ok() {
```

Comment:
не потрібно писати test на початку, це зрозуміло ітак що це тестю (і так само в інших тестах)
```suggestion
    public void reduceProcess_Ok() {
```